src/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,148 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import omit from 'lodash.omit';
+
+import { Button, ButtonIcon } from 'react-lds';
+
+export default class Accordion extends React.Component {
+  static defaultProps = {
+    className: null,
+    styled: false,
+  }
+
+  static propTypes = {
+    /**
+     * class name
+     */
+    className: PropTypes.string,
+    /**
+     * array of sections
+     */
+    sections: PropTypes.arrayOf(PropTypes.shape({
+      summary: PropTypes.string.isRequired,
+      id: PropTypes.string.isRequired,
+      content: PropTypes.node.isRequired,
+      summaryOnClick: PropTypes.func,
+    })).isRequired,
+    /**
+      * which section should be open by default, defaults to first
+      */
+    defaultOpen: PropTypes.string,
```

Comment:
This only works as an uncontrolled component, so there is no way to interface with this component from outside. You could consider adding an `open`-prop that directly sets the open section and disregards internal state in that mode.